airflow/ui/src/components/TriggerDag/TriggerDag.tsx

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export type DagParams = {
+  configJson: Record<string, string>;
+  dagId: string;
+  logicalDate: string;
+  runId: string;
+};
+
+export const TriggerDag = (dagParams: DagParams) => {
+  // TODO triggering logic
+  console.log("Triggering DAG with parameters:", dagParams);
```

Comment:
Just checked console log - seems currently empty/default values are logged, irrespective what I enter on the form

Reply:
Yaa, fixing that

Reply:
Thanks for making it an alert() - this looks good now and values are carried over there!